security/azure/pg_conn_throttling/pg_conn_throttling.pt

```diff
@@ -0,0 +1,193 @@
+name "Azure Ensure PostgreSQL Servers Connection Throttling Enabled"
+rs_pt_ver 20180301
+type "policy"
+short_description "Report if any PostgreSQL server instances do not have connection throttling enabled. \n See the [README](https://github.com/flexera/policy_templates/tree/master/security/azure/pg_conn_throttling) and [docs.rightscale.com/policies](https://docs.rightscale.com/policies/) to learn more."
```

Comment:
Shouldn't the doc's link be changed from rightscale to flexera "[docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."

Reply:
It should but I will be doing a mass fix on this once these pull requests are completed.